README.md

```diff
@@ -1,2 +1,37 @@
-# inji-vci-client-ios-swift
-As swift package manager recognises the library from github repo instead of submodules, We need to create new repo for vci client ios library.
+
+# VCI Client
+- Request credential to Credential issuer
+
+## Installation
+- Clone the repo
+- In your swift application go to file > add package depenedecy > add the  https://github.com/tw-mosip/inji-vci-client-ios-swift.git in git search bar> add package
+- Import the library and use
+
+## APIs
+
+##### Request Credential
+
+Request for credential from the issuer, and receive the credential reponse back in string.
+
+```
+    let requestCredential = try await VCIClient().getCredential(issuerMeta: issuer, signer: signer, accessToken: accessToken, publicKey: publicKeyPEM)
```

Comment:
method name here not matching with actual implementation.
It has to be requestCredential.